city_scrapers/spiders/cook_county_board_ethics.py

```diff
@@ -0,0 +1,151 @@
+import re
+from datetime import datetime, timedelta
+
+import scrapy
+from city_scrapers_core.constants import (
+    ADVISORY_COMMITTEE, BOARD, COMMISSION, COMMITTEE, NOT_CLASSIFIED
+)
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class CookCountyBoardEthicsSpider(CityScrapersSpider):
+    name = 'cook_county_board_ethics'
+    agency = 'Cook County Government Board of Ethics'
+    timezone = 'America/Chicago'
+    allowed_domains = ['www.cookcountyil.gov']
+
+    def start_requests(self):
+        today = datetime.now()
+        next_month = today.replace(day=28) + timedelta(days=5)
+        for dt in [today, next_month]:
+            mo_str = dt.strftime('%Y-%m')
+            url = 'https://www.cookcountyil.gov/calendar-node-field-date/month/{}'.format(mo_str)
+            yield scrapy.Request(url=url, method='GET', callback=self.parse)
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for url in self._get_event_urls(response):
+            yield scrapy.Request(url, callback=self._parse_event, dont_filter=True)
+
+    def _parse_event(self, response):
+        """Parse the event page."""
+        title = self._parse_title(response)
+        meeting = Meeting(
+            title=title,
+            description=self._parse_description(response),
+            classification=self._parse_classification(title),
+            start=self._parse_start(response),
+            end=self._parse_end(response),
+            time_notes='',
+            all_day=self._parse_all_day(response),
+            location=self._parse_location(response),
+            links=self._parse_links(response),
+            source=response.url,
+        )
+        meeting['id'] = self._get_id(meeting)
+        meeting['status'] = self._get_status(meeting)
+        return meeting
+
+    def _get_event_urls(self, response):
+        """
+        Get urls for all Board of ethics meetings on the page.
+        """
+        return [
+            response.urljoin(href) for href in
+            response.xpath('//a[contains(@href, "board-ethics")]').css('a::attr(href)').extract()
```

Comment:
It might be safer to check the text than the URL here. Either way works, but we've seen some strange URL issues on sites before